addons/l10n_cl/models/l10n_latam_document_type.py

```diff
@@ -42,3 +42,6 @@ def _is_doc_type_export(self):
 
     def _is_doc_type_electronic_ticket(self):
         return self.code in ['39', '41'] and self.country_id.code == 'CL'
+
+    def _is_doc_type_refund(self):
```

Comment:
I wonder, why not just add code '61' in the definition of `_is_doc_type_vendor` above?